sndev

```diff
@@ -441,6 +478,9 @@ COMMANDS
     fund          pay a bolt11 for funding
     withdraw      create a bolt11 for withdrawal
 
+  cln:
```

Comment:
Maybe this should be a flag to `fund` instead, and the help section renamed to `wallet` rather than `lnd`?

Reply:
We'll just leave this for now I think.

Reply:
> Maybe this should be a flag to fund instead, and the help section renamed to wallet rather than lnd

Ah yes, this would be better